test/functional/interface_bitcoin_cli.py

```diff
@@ -25,15 +25,6 @@ def run_test(self):
         """Main test logic"""
         self.nodes[0].generate(BLOCKS)
 
-        cli_response = self.nodes[0].cli("-version").send_cli()
-        assert "{} RPC client version".format(self.config['environment']['PACKAGE_NAME']) in cli_response
-
-        self.log.info("Compare responses from getwalletinfo RPC and `bitcoin-cli getwalletinfo`")
-        if self.is_wallet_compiled():
-            cli_response = self.nodes[0].cli.getwalletinfo()
-            rpc_response = self.nodes[0].getwalletinfo()
-            assert_equal(cli_response, rpc_response)
-
```

Comment:
Why this test removed?

Reply:
Not removed; moved to line 75 `assert_equal(self.nodes[0].cli.getwalletinfo(), wallet_info)`

Reply:
which avoids calling rpc getwalletinfo twice, a conditional, and a few lines of code.

Reply:
Ok, yes, I didn't read the whole block correctly, assumed it checks `-version`, which is also checked below, just that it outputs, not what it outputs. And it's `bitcoin-cli` version anyways, can differ from version returned by `-getinfo`, nothing to compare against there.

Reply:
@kristapsk thanks for reviewing! Can you ACK?